packages/loot-core/src/server/aql/schema/index.ts

```diff
@@ -61,6 +61,7 @@ export const schema = {
     name: f('string', { required: true }),
     transfer_acct: f('id', { ref: 'accounts' }),
     tombstone: f('boolean'),
+    favorite: f('boolean'),
```

Comment:
:clap: **Praise**: Thanks for remembering this part.  It usually gets forgotten.